internal/x/format/base_visitor.go

```diff
@@ -138,12 +138,15 @@ func (v *baseVisitor) POptions(isFieldOption bool, options ...*proto.Option) {
 		if len(o.Constant.Array) == 0 && len(o.Constant.OrderedMap) == 0 {
 			v.PWithInlineComment(o.InlineComment, prefix, o.Name, ` = `, o.Constant.SourceRepresentation(), suffix)
 		} else if len(o.Constant.Array) > 0 { // both Array and OrderedMap should not be set simultaneously, need more followup with emicklei/proto
-			// TODO https://github.com/uber/prototool/issues/59
+			v.Failures = append(
+				v.Failures,
+				text.NewFailuref(o.Position, "INVALID_PROTOBUF", "top-level options should never be arrays, this should not compile with protoc"),
```

Comment:
should not or cannot?

Reply:
Cannot, to the best I can figure out. However, it's a valid "switch" here, so I want to record it as a failure if we somehow hit it, as opposed to just ignoring it.

Reply:
Whoops, already merged. This error message should never print so it shouldn't (can't :-) ) be a big deal, but I can do a subsequent PR if it's a problem.